tabs-tabs-tabs/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "applications": {
+    "gecko": {
+      "id": "tabs-tabs-tabs@mozilla.org",
+      "strict_min_version": "47.0.0"
```

Comment:
Yes, but I'd guess that it thinks (correctly) that 45.0.0 < 47.0a1 < 47.0.0.

So if Nightly is 47.0a1, then 45.0.0 installs fine on Nightly, but 47.0.0 does not. But this add-on can't use 45.0.0, because it needs tabs.duplicate().

"47.0a1" will work for now, and I think it will still work when 47 is 47.0. If not we can change it then.

This makes me think we need better docs for version numbers...